resources/sample_qc.py

```diff
@@ -1,12 +1,20 @@
 
 
 # Sample QC files
-def qc_mt_path(data_type: str):
-    return 'gs://gnomad/sample_qc/mt/gnomad.{}.high_callrate_common_biallelic_snps.mt'.format(data_type)
+def qc_mt_path(data_type: str, pruned: bool = False):
+    # can be joint, pruned only for joint
+    return 'gs://gnomad/sample_qc/mt/gnomad.{}.high_callrate_common_biallelic_snps{}.mt'.format(data_type,
```

Comment:
Let's put in `{0}` and `{1}` to make the assignments absolutely clear

Reply:
Really? I don't love those when its not necessary, but I'm happy to be voted down. TBH my favorite is:
```
return 'gs://gnomad/sample_qc/mt/gnomad.{data_type}.high_callrate_common_biallelic_snps{part}.mt'.format(data_type=data_type, part=part)
```
or the new python3 formatting:
```
return f'gs://gnomad/sample_qc/mt/gnomad.{data_type}.high_callrate_common_biallelic_snps{part}.mt'
```

Reply:
I don't particularly mind which format we pick (the other two options look good to me), but I like having the assignments clearer than just scanning across the line to count which thing is what